attiny/attiny.ino

```diff
@@ -125,35 +133,40 @@ void requestEvent() {
 void receiveEvent(uint8_t howMany) {
   if (howMany < 1) {
     // Message too short.
-    blinks = 2;
-    return;
-  }
-  if (howMany > 10) { // Message too long. Read bytes then exit.
-    while(TinyWireS.available()) {
-      TinyWireS.receive();
-    }
-    blinks = 3;
+    setBlinks(10);
     return;
   }
-  byte message[10];
-  int i = 0;
-  while(TinyWireS.available()) {
-    message[i++] = TinyWireS.receive();
-  }
-  switch(message[0]) {
+  bool successfullMessageRead = false;
```

Comment:
"successful" has one "l". Also, the name is a bit long. What about `successfulRead` or `messageOk`?